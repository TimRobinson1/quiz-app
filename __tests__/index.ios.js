import { Text } from 'react-native';
import React from 'react';
import QuizApp from '../index.ios.js';
import { shallow } from 'enzyme';
// import { expect } from 'chai';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it("renders correctly", () => {
  const tree = renderer.create(<QuizApp />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("has content", () => {
  const page = shallow(<QuizApp />);
  // console.log(page);
  expect(page.find('.texttag').node.props.children).toContain('Hello World!');
})
