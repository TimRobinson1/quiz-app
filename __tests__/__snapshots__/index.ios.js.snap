// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    className="texttag"
    ellipsizeMode="tail"
  >
    Hello World!
  </Text>
</View>
`;
